site/docs/components/progress-bar.mdx

```diff
@@ -37,11 +38,8 @@ import WhenNotToUse from "docs-components/WhenNotToUse"
 - Static process or loading progress:
     - **Static**: a static progress bar shows current progress through a process that will not progress without user interaction.
     - **Loading**: a loading progress bar shows current progress through a loading process that the system will progress by itself without user interaction. We use animation to show that the system is operating.
-- Props:
-    - Min
-    - Max
-    - Initial value
-    - Infinite/indeterminate
+- render a Percentage (33%) or Fraction (1/10)
+- Infinite/indeterminate (not yet designed)
```

Comment:
I updated the docs slightly to reflect how I implemented it. I think it's better to keep these docs high level design guides rather than an exact spec of the React props (so I removed the "props" label) - it's too easy for it to get out of sync. We can automatically generate React/Typescript props docs.